backend/prisma/migrations/20240702184800_posts_image_tables/migration.sql

```diff
@@ -0,0 +1,26 @@
+-- CreateTable
+CREATE TABLE "Post" (
+    "id" SERIAL NOT NULL,
+    "title" TEXT NOT NULL,
+    "content" TEXT NOT NULL,
```

Comment:
_nit_: best to add some character limits

Reply:
added character limits using @db.VarChar()